afs-quota-notify

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/env python
-# (ignore bad script name) pylint: disable=C0103
+#!/usr/bin/env python2
+# ^^^ TODO convert me (move off of ingwe first)
```

Comment:
are there python2-only-isms in this script?

Reply:
I _think_ just the imports -- ConfigParser, StringIO, maybe email or smtplib.